common/script/ops/webhooks.js

```diff
@@ -0,0 +1,152 @@
+import refPush from '../libs/refPush';
+import validator from 'validator';
+import i18n from '../i18n';
+import { v4 as uuid } from 'uuid';
+import {
+  BadRequest,
+  NotFound,
+} from '../libs/errors';
+
+const TASK_ACTIVITY_DEFAULT_OPTIONS = Object.freeze({
```

Comment:
I think we should make using `Object.freeze` standard anywhere we are creating constant that is an object or an array.